docs/knowledge/catalogs/test-catalog/text_generation/index.rst

```diff
@@ -4,10 +4,16 @@ Text generation tests
 - **Injections**
 
   - :func:`~.giskard.testing.tests.llm.test_llm_char_injection`
-  - :func:`~.giskard.testing.tests.llm.test_llm_prompt_injection`
+  - :func:`~.giskard.testing.tests.llm.test_llm_single_output_against_strings`
+  - :func:`~.giskard.testing.tests.llm.test_llm_output_against_strings`
 
 - **LLM-as-ajudge**
 
-  - :func:`~.giskard.testing.tests.llm.test_llm_output_against_requirement`
-  - :func:`~.giskard.testing.tests.llm.test_llm_single_output_against_requirement`
   - :func:`~.giskard.testing.tests.llm.test_llm_output_coherency`
+  - :func:`~.giskard.testing.tests.llm.test_llm_output_plausibility`
+  - :func:`~.giskard.testing.tests.llm.test_llm_output_against_requirement_per_row`
+  - :func:`~.giskard.testing.tests.llm.test_llm_single_output_against_requirement`
+  - :func:`~.giskard.testing.tests.llm.test_llm_output_against_requirement`
+  - :func:`~.giskard.testing.tests.llm.test_llm_ground_truth`
```

Comment:
@rabah-khalek can you put a new section header for ground truth tests since theyre not llm as a judge tests

Reply:
you're right!